x-pack/docs/en/security/authentication/built-in-users.asciidoc

```diff
@@ -16,6 +16,10 @@ used to <<set-built-in-user-passwords,set all of the built-in user passwords>>.
 storing monitoring information in {es}. It has the `remote_monitoring_agent` and 
 `remote_monitoring_collector` built-in roles. 
 
+TIP: The built-in users serve a specific purpose and are not intended for every
+day use. In particular, do not use the `elastic` superuser unless full access to
```

Comment:
I worry about someone interpreting "not intended for every day use" to mean that they shouldn't use the `kibana` user in their `kibana.yml`.
I don't have a better suggestion though.

Reply:
Good point, I've switched to "general use" instead of "every day use".